.internal-ci/helm/mobilecoind/templates/mobilecoind-json-service.yaml

```diff
@@ -14,3 +14,6 @@ spec:
   - name: json
     port: 9090
     targetPort: json
+  - name: json-http
+    port: 80
```

Comment:
Why add this port?

Reply:
Nice catch. The idea was to use port 80 on the service so that calling services don't need to specify a port. Generally I think we should prefer to use port `80` on services to abstract that away from the client. I think that preferable than trying to match the internal port of the pod and exposing that implementation detail to clients

Reply:
We can map ports in 2 ways:
1. use a single hostname and define multiple ports.
2. use a hostname per service and use a single standard port for the service (80, 443, 5432...) 

I think this is the right approach, using the second where we just point at a sane abstracted service name and use the standard port.

Reply:
This does get a little bit fuzzy where a pod might be listening on multiple "standard" ports.  Like metrics on "9090".  

I think in general where an app is publishing an "HTTP" (or GRPC) port, mapping it to 80 on the k8s "Service" load-balancer makes sense and can ease the configuration burden